kernel/format.c

```diff
@@ -479,6 +483,26 @@ PHP_METHOD(vtiful_format, borderColorOfTheFourSides)
 }
 /* }}} */
 
+/** {{{ \Vtiful\Kernel\Format::rotation(int $angle)
+ */
+PHP_METHOD(vtiful_format, rotation)
+{
+    zend_long angle = 0;
+
+    ZEND_PARSE_PARAMETERS_START(1, 1)
+            Z_PARAM_LONG(angle)
```

Comment:
angle range: rotation angle in the range -90 to 90 and 270.

Reply:
Patched it.